tigerpath/majors_and_certificates/majors/MAE.json

```diff
@@ -259,9 +261,9 @@
           "req_list": [
             {
               "name": "Engineering Science Courses",
-              "max_counted": 5,
-              "min_needed": 5,
-              "explanation": "Five Engineering Science courses are required",
+              "max_counted": 6,
```

Comment:
https://mae.princeton.edu/sites/default/files/MAE-UG%20HANDBOOK%20AY%202021-2022%20Classes%202022%20-2025%20for%20Web%20April%202021.pdf

"Engineering Science Courses – In the Aerospace Program six courses are required and
in the Mechanical Program five courses are required from Sections (a) (b) and (c). Core
requirements for both programs are footnoted."